### PR TITLE
Update badge captions

### PR DESCRIPTION
**Pull request by balloob**
Use `name` as source for badge caption and add some overrides where useful.

localhost_5000_faq_ (2)

### CODE REVIEW

File: redirect.json [hunk 1]
```diff
@@ -248,14 +262,14 @@
...
-    "badge": "Z-Wave JS Configuration",
+    "badge": "Z-Wave JS",
     "description": "show your Z-Wave JS configuration panel",
     "introduced": "2021.3"
   },
   {
     "redirect": "config_zha",
     "name": "ZHA Configuration",
-    "badge": "ZHA Configuration",
+    "badge": "ZHA",
```
Comment (bramkragten) [inline redirect.json:272]: Was thinking this should probably be Zigbee?

File: redirect.json [hunk 2]
```diff
@@ -151,19 +157,21 @@
   {
     "redirect": "lovelace_dashboards",
     "name": "Lovelace: Dashboards",
+    "badge": "Lovelace",
```
Comment (bramkragten) [inline redirect.json:160]: This looks like the lovelace panel, and not dashboard config?

Comment (balloob) [reply redirect.json:160]: hmm true. Make it Lovelace Config maybe? Dashboard is ambiguous too.

File: redirect.json (hunk 1 above)
Comment (balloob) [reply redirect.json:272]:
```suggestion
    "badge": "Zigbee",
```

File: redirect.json (hunk 2 above)
Comment (balloob) [reply redirect.json:160]:
```suggestion
    "badge": "Lovelace Config",
```

File: redirect.json [hunk 3]
```diff
@@ -151,19 +157,21 @@
   {
     "redirect": "lovelace_dashboards",
     "name": "Lovelace: Dashboards",
+    "badge": "Lovelace Config",
     "description": "show your Lovelace dashboards",
```
Comment (bramkragten) [inline redirect.json:161]: Also update this one to match?

Comment (bramkragten) [reply redirect.json:161]: Show your lovelace dashboards configuration

Comment (balloob) [reply redirect.json:161]:
```suggestion
    "description": "show your Lovelace dashboard config",
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**